src/AutoBuild.jl

```diff
@@ -894,7 +894,8 @@ function rebuild_jll_package(obj::Dict;
                              gh_org::String = "JuliaBinaryWrappers",
                              verbose::Bool = false,
                              lazy_artifacts::Bool = false,
-                             from_scratch::Bool = true)
+                             from_scratch::Bool = true,
+                             from_yggdrasil::Bool = false)
```

Comment:
Is it too magical to find if there's an enclosing `git` repository, check its remote URL, and auto-set `from_yggdrasil`?  ;). It's fine to say "yes", this works fine since we fully control how this method is called.